.changeset/ten-grapes-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+`aria-orientation` attribute removed from ButtonGroup to meet accessibility requirements.
```

Comment:
I would mention this in the PR description

Reply:
Done!